cuda_core/cuda/core/experimental/_launcher.py

```diff
@@ -53,13 +58,22 @@ class LaunchConfig:
 
     # TODO: expand LaunchConfig to include other attributes
     grid: Union[tuple, int] = None
+    cluster: Union[tuple, int] = None
     block: Union[tuple, int] = None
     stream: Stream = None
     shmem_size: Optional[int] = None
 
     def __post_init__(self):
+        _lazy_init()
         self.grid = self._cast_to_3_tuple(self.grid)
         self.block = self._cast_to_3_tuple(self.block)
+        # thread block clusters are supported starting H100
+        if self.cluster is not None:
+            if not _use_ex:
+                raise CUDAError("thread block clusters require cuda.bindings & driver 11.8+")
+            if Device().compute_capability < (9, 0):
+                raise CUDAError("thread block clusters are not supported below Hopper")
```

Comment:
I personally think this message should refer to compute capability rather than archictecture name chronology

Reply:
Right let us assume >=9.0 is the correct thing to do. If we later find out it is not, we can make changes.

Reply:
I agree with Keenan, explicitly stating the compute capability in the error message is a more direct way to communicate what user should be programmatically checking before using this functionality. Calling for `Device().compute_capability` is easier than comparing against a list of architecture names.